core/domain/exp_services.py

```diff
@@ -1799,6 +1799,12 @@ def are_changes_mergeable(exp_id, frontend_version, change_list):
         # related (affected by or affecting) customization args. This list
         # can be changed when any new property is added or deleted which
         # affects or is affected by customization args.
+        # This list is similar to the other two lists just below this but
+        # don't merge them or make them one because they are for different
+        # properties so if in the future any new property is added which
+        # is affected by cust_args only then in that case we'll add that
+        # property name in properties_related_to_cust_args only.
+        # So we should keep a common list.
```

Comment:
Instead of this, maybe at the top you can say "Note: List X, Y, Z should remain separate lists irrespective of its content. This is because <explanation about why they are closely associated to a property>."?

".. other two lists .." is a bit vague. Avoid refering to things in a relative manner (like "just below") because it is possible that the position of the variables change.

Reply:
Agreed. Done.